CHANGELOG.md

```diff
@@ -7,14 +7,15 @@ and this project adheres to Rust's notion of
 
 ## [Unreleased]
 
+## [0.13.1] - 2022-07-05
 ### Added
 - `bellman::groth16::batch::Verifier` now has a `verify_multicore` method (when
   the `multicore` feature is enabled) which will internally use the global rayon
   thread pool to parallelize the verification of a batch of proofs.
 
 ### Changed
-- The `multicore` feature now requires the `getrandom` feature of the
-  `rand_core` crate.
+- The `multicore` feature now enables the `getrandom` feature of the `rand_core`
+  crate.
```

Comment:
I'm still on the fence about this, as it is a breaking change to anyone trying to use `bellman` on a platform that [`getrandom` doesn't support](https://docs.rs/getrandom/0.2.7/getrandom/#supported-targets). But it seems like there will be very few platforms that don't have a system RNG but _do_ support allocators, threads, and the standard library.

Reply:
The [register_custom_getrandom!](https://docs.rs/getrandom/0.2.7/getrandom/macro.register_custom_getrandom.html) macro can be used to register an RNG implementation, although I agree that would be a breaking change on such platforms.

Reply:
Still, in this case the benefit of a point release outweights this I think; if someone complains then we can deal with it then.